### ISSUE TITLE
Problems related to model performance

### ISSUE THREAD

**Issue by JwFbupt** [MCG-NJU/TDN#51]
Hi~
I have trained a TDN(ResNet50+8frame) on something-v1 as well as something-v2. However, there exists some gaps between the results we do and the reported one. On something-v2, best Prec@1 is only 61.567. 
We follow the command and process that you mentioned in README.md.  I wonder that maybe some hyper-parameters settings in our experiments is not the same as your experiments.
<img width="1086" alt="截屏2022-01-19 下午3 12 31" src="https://user-images.githubusercontent.com/49198978/150081797-bf278f9c-100f-4641-8819-03930a7d3238.png">

**Comment by yztongzhan** [comment]
Hi @JwFbupt! Thanks for your question! Please provide your command here, so I could help you.

**Comment by JwFbupt** [comment]
bash train_wsod.sh 0,1,2,3,4,5,6,7 pascal_voc vgg16 voc07_wsddn_pre lbba_init.pth

**Comment by JwFbupt** [comment]
rlaunch --cpu=48 --gpu=8 --memory=$((1024*140))  --max-wait-time=12h --charged-group=v_event  -- python -m torch.distributed.launch --master_port 12347 --nproc_per_node=8 \
                main.py  somethingv2  RGB --arch resnet50 --num_segments 8 --gd 20 --lr 0.01 \
                --lr_scheduler step --lr_steps  30 45 55 --epochs 60 --batch-size 8 \
                --wd 5e-4 --dropout 0.5 --consensus_type=avg --eval-freq=1 -j 4 --npb

**Comment by yztongzhan** [comment]
Hi @JwFbupt! For your convenience, we provide the [training log](https://drive.google.com/file/d/18Pfiw-f-hyaQ9F_R3TYH8nwu0W6g2Mj_/view?usp=sharing) and [tensorboard file](https://drive.google.com/file/d/1qjrZtgRv9HuNmNI2GDiBTVZwIFjxaG4E/view?usp=sharing) for SSV1 TDN-50. We hope these could be helpful for you.

**Comment by yztongzhan** [comment]
Hi @JwFbupt! The command is right. Could you share the training log? 
For your convenience, we provide the [training log](https://drive.google.com/file/d/1bkBLoJkDqp4kEuaPAcQLGS94TYFBzgl4/view?usp=sharing) and [tensorboard file](https://drive.google.com/file/d/1p5jFJmO9936QyIa1WmjKeIGcHP_Cfkq7/view?usp=sharing) for TDN-R50-Segment-8 on SSV2. We hope these could be helpful for you.

**Comment by yztongzhan** [comment]
Hi @JwFbupt! Could you share your training log? Or, you have addressed this problem? 
I will close this issue if you have no reply in two weeks.

**Comment by yztongzhan** [comment]
It seems no problem in the training log. Could you reproduce the results of other method like TSM and TEA on your SSV2?

**Comment by yztongzhan** [comment]
Besides, I want to know if you modify the validation code by yourself?

**Comment by JwFbupt** [comment]
I have reproduced TSM on somethingv2, and obtained promising results. In this version, we did not modify any codes.
Thank you for your reply, and I will try to solve the problem.

**Comment by JwFbupt** [comment]
hi @yztongzhan, we have reproduce the results of TEA as well as TSM on sthv2.
I would like to know, whether the official model use mixup method or flip on sthv2

**Comment by yztongzhan** [comment]
We use **flip aug** for SSV2 training,  which also support in [MMaction2 for such TSM](https://github.com/open-mmlab/mmaction2/blob/master/configs/recognition/tsm/tsm_r50_flip_1x1x8_50e_sthv1_rgb.py). But we want to point out that our `label generation script` is different from MMaction2, you need to check our label carefully.
I hope these could be useful for you.
